src/Derivative.cpp

```diff
@@ -1365,8 +1330,8 @@ void ReverseAccumulationVisitor::propagate_halide_function_call(
     // We then substitute the original variable names back to get
     // f'(x, y, z) += g'(x, x + 1, z - 1)
     //
-    // Currently we don't want to mess with system solving yet,
-    // So we gather all arguments that contains multiple pure variables,
+    // Currently we don't want to mess with system solving yet.
```

Comment:
Nit: "Currently... yet" is redundant. Maybe omit the "yet" at the end.

Reply:
Fixed.